examples/server_client/test_suites/base_tests_multiple_iter_parallel.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2014 LinkedIn Corp.
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import time
+
+import perf
+import zopkio.runtime as runtime
+import zopkio.test_utils as testutilities
+
+
+SAMPLE = lambda size: [str(i) for i in range(1, size + 1)]
```

Comment:
what is the objective here? if you want to generate the samples I would use a regular function rather than a lambda, and use xrange not range